docs/modules/user-guide/partials/proc_adding-plugin-component-to-a-devfile.adoc

```diff
@@ -41,47 +41,6 @@ This section describes how to add a `plugin` component to a devfile.
 ----
 ====
 
```

Comment:
Again sorry, lines were not edited

1. `Define the id attribute. It is slash-separated publisher, name and version `
This is still being worked on and is perhaps not the same as devfile v1. @yangcao77 can you pls shed some light since you're currently working on this format?

2. `more information about registry can be found in the {prod-short} plug-in registry GitHub repository.`
this is no longer true, we dont have one atm

FWIW, plugin is removed from devfile v2 for the next release but the discussion is valid for parents.

Reply:
plugin is going to be removed in devfile v2

for parent registry in V2, the id is the stack name, so it is expected to find devfile by sending GET request to `{registryUrl}/devfiles/{id}`, as proposed in https://github.com/devfile/api/blob/master/docs/proposals/registry/index-server-REST-API.md